_includes/neurips/2019/about.neurips.2019.html

```diff
@@ -97,13 +98,35 @@ <h3>Platinum</h3>
 		  <a target="_blank" href="https://www.google.com"><img width=500
```

Comment:
- Please add missing sponsor Facebook under platinum.
- Deepmind should be listed under Silver, not Gold.

Reply:
I did the images to the nips folder